src/Ztm.WebApi/AddressPools/ReceivingAddress.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using NBitcoin;
+
+namespace Ztm.WebApi.AddressPools
+{
+    public sealed class ReceivingAddress
+    {
+        public Guid Id { get; }
+        public BitcoinAddress Address { get; }
+        public bool IsLocked { get; }
+        public List<ReceivingAddressReservation> ReceivingAddressReservations { get; }
+
+        public ReceivingAddress(Guid id, BitcoinAddress address, bool isLocked, List<ReceivingAddressReservation> receivingAddressReservations)
+        {
+            this.Id = id;
+            this.Address = address;
+            this.IsLocked = isLocked;
+            this.ReceivingAddressReservations = receivingAddressReservations;
+        }
+
+        public bool Available
```

Comment:
We should remove this property. The problem is it will cause more way to check if it available or not, which is not consistent.